core/src/test/java/org/frankframework/util/XsltStreamingTest.java

```diff
@@ -156,7 +157,7 @@ public void testStreamingXsltViaStreamSource() throws TransformerException, IOEx
 		ByteArrayInputStream bais = new ByteArrayInputStream(input.toString().getBytes());
 		Source source = new StreamSource(new LoggingInputStream(bais, sc));
 
-		tp.transform(source, result);
+		tp.transform(source, result, (Map<String,Object>) null);
```

Comment:
Kan je hier misschien `Collections.emptyMap()` gebruiken?

Reply:
Nee, this is vanwege overloading, hij moet een bepaalde methode aanroepen, en niet met iets anders dan waarde `null`.